Makefile

```diff
@@ -2,35 +2,22 @@
 
 CURDIR := $(shell pwd)
 
-KEEP_FILE := '**/*.proto,**/*.sh'
-
-export PATH := $(CURDIR)/_vendor/bin:$(PATH)
-export GOPATH := $(CURDIR)/_vendor
+export PATH := $(CURDIR)/bin/:$(PATH)
 
 all: go rust
 
-go:
+init:
+	mkdir -p $(CURDIR)/bin
+go: init
 	# Standalone GOPATH
 	./generate_go.sh
 	go build ./pkg/...
 
-rust:
+rust: init
 	./generate_rust.sh
 	cargo check
 
 update_go_pkg:
```

Comment:
better to use a hyphen `-` instead of an underscore `_`?
https://style-guides.readthedocs.io/en/latest/makefile.html